src/ui/ui_settings/ui_settings_service.js

```diff
@@ -1,8 +1,6 @@
 import { defaultsDeep, noop } from 'lodash';
 import { errors as esErrors } from 'elasticsearch';
 
-import { getDefaultSettings } from './defaults';
-
 function hydrateUserSettings(userSettings) {
   return Object.keys(userSettings)
     .map(key => ({ key, userValue: userSettings[key] }))
```

Comment:
IIRC I was coming down from an rx/observable high and didn't like that this method was previously implemented as a reduce that just assigned to a value and returned it (so that it could filter that value)... Then I remember wanting to implement it without the `if()`... Yeah, not a huge fan of how it turned out. If you want to take a stab at it in another pr I'd love to see your prefered alternative